wallet-new/src/Cardano/Wallet/API/Response.hs

```diff
@@ -43,36 +45,22 @@ deriveJSON defaultOptions { constructorTagModifier = map Char.toLower . reverse
 instance Arbitrary ResponseStatus where
     arbitrary = elements [minBound .. maxBound]
 
--- | An `ExtendedResponse` allows the consumer of the API to ask for
--- more than simply the result of the RESTful endpoint, but also for
--- extra informations like pagination parameters etc.
-data ExtendedResponse a = ExtendedResponse
-  { extData   :: a
+-- | An `WalletResponse` models, unsurprisingly, a response (successful or not)
+-- produced by the wallet backend.
+-- Includes extra informations like pagination parameters etc.
+data WalletResponse a = WalletResponse
```

Comment:
Record fields do not follow conventions here and thus prone to clash on import (..).